### PR TITLE
Add support for Haven (XHV) using CryptoNight Heavy

### PR DESCRIPTION
**Pull request by havenprotocol**
Haven just hardforked to cryptonight heavy and needs miner support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Besides [Monero](https://getmonero.org), following coins can be mined using this
 - [Intense](https://intensecoin.com)
 - [Karbo](https://karbo.io)
 - [Sumokoin](https://www.sumokoin.org)
+- [Haven](https://havenprotocol.com)
```
Comment (fireice-uk) [inline README.md]: This list is in alphabetical order, please insert at the proper position.

**Comment by fireice-uk** [comment]
Can you please link to a major Haven pool? Our criteria for inclusion is at least 1MH/s of sustained hashrate.

**Comment by felipe-vvoosh** [comment]
@fireice-uk, this one is very close to 1 MH/s:
https://xhv.luckypool.io/

File: README.md (hunk 1 above)
Comment (havenprotocol) [reply README.md]: Have updated the ordering to be alphabetical

**Comment by havenprotocol** [comment]
Just hardforked so a lot of miners are still coming back online or trying to get working miners. Pre hardfork (using cryptonight) this pool was around 15MH/sec consistently.

https://xhv.luckypool.io/

**Comment by fireice-uk** [comment]
Thanks!

**Comment by psychocrypt** [comment]
Note haven will not work for AMD gpus. The reason is that the amd code
contains the version ckeck and forkes only with block version 3.
I will fix it the next days.

**Comment by felipe-vvoosh** [comment]
Thanks @psychocrypt!

**Comment by romin991** [comment]
so i can't mine haven with amd Vega?

**Comment by felipe-vvoosh** [comment]
Don't be lazy...

**Comment by STRATZ-Ken** [comment]
@psychocrypt Any update on this?

**Comment by psychocrypt** [comment]
The PR to fix it are in the pipeline (opened)

p96xl <notifications@github.com> schrieb am Di., 17. Apr. 2018, 01:23:

**Comment by STRATZ-Ken** [comment]
Yup. Did that.  Also reset all bios settings. Both didn't help

Sent from my iPhone

On Apr 18, 2018, at 6:23 PM, psychocrypt <notifications@github.com<mailto:notifications@github.com>> wrote:

please remove your amd.txt and let the miner create the config for you. You
need 4MB per hash dor heavy

-

**Pull request merged**